# Generated for NaoQi method ALExtractor.getCurrentPrecision
# Gets the current precision.

---
# Precision of the extractor.
float32 value


